src/main/java/com/pmm/budgetcalculator/expensing/controller/ExpenseTypeController.java

```diff
@@ -0,0 +1,40 @@
+package com.pmm.budgetcalculator.expensing.controller;
+
+import com.pmm.budgetcalculator.expensing.entity.ExpenseType;
+import com.pmm.budgetcalculator.expensing.service.ExpenseTypeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("expenses/expense_types")
+public class ExpenseTypeController {
+    private final ExpenseTypeService expenseTypeService;
+
+
+//    @GetMapping
+//    public ResponseEntity<List<ExpenseType>> getMany(
+//            @RequestParam(value = "expenseType", required = false) String expenseType) {
+//        return ResponseEntity.ok(expenseTypeService.findByCriteria(expenseType));
+//    }
+    @GetMapping
+    public ResponseEntity<List<ExpenseType>> getAllExpenseTypes() {
+        return ResponseEntity.ok(expenseTypeService.getAllExpenseTypes());
+    }
+
+
+    @PostMapping
+    public ResponseEntity<ExpenseType> createExpenseType(@RequestBody ExpenseType expenseType) {
+        return ResponseEntity.ok(expenseTypeService.createExpenseType(expenseType));
+    }
+
+    @DeleteMapping("/{id}")
+    public void removeExpenseType(@PathVariable("id") Long id) {
```

Comment:
Something I missed om expenseController and please fix it there as well.                     You should never return void from controller. Return ResponseEntity<Void> in case of delete request.                                    Why?                                                                                                       1. It's less confusing when all methods return ResponseEntity                                           2. In ResponseEntity you can specify what HTTP code you want to respond when you need to.